backends/platform/libretro/dependencies.mk

```diff
@@ -95,6 +97,7 @@ OBJS_DEPS += $(DEPS_PATH)/$(DEPS_FOLDER_libretro-deps)/fluidlite/src/fluid_chan.
 	$(DEPS_PATH)/$(DEPS_FOLDER_libretro-deps)/fluidlite/src/fluid_tuning.o \
 	$(DEPS_PATH)/$(DEPS_FOLDER_libretro-deps)/fluidlite/src/fluid_voice.o \
 	$(DEPS_PATH)/$(DEPS_FOLDER_libretro-deps)/fluidlite/src/fluid_init.o
+OBJS_DEPS += $(OBJS_DEPS_FLUIDSYNTH)
```

Comment:
Do you really need this?
It seems that `OBJ_DEPS_FLUIDSYNTH` will be dragged from `libdeps_fluidsynth.a` which is itself dragged from `$(LIBDEPS_ARCHIVES)` and  `$(TARGET)`.
Maybe it has some other use though.

Reply:
Yes, for `make clean`

Reply:
My thinking was to keep `OBJS_DEPS` and avoid having if statements for `make clean`. But `make clean` calls `rm -f` which doesn't care if the files don't exist, so we could drop `OBJS_DEPS` and just rm -f all the dependency vars